pkg/releasenotes/releasenotes.go

```diff
@@ -83,19 +114,12 @@ func (c *Client) Get(org, repo, branch, milestone string) (ReleaseNotes, *Statis
 	}
 	for _, p := range prs {
 		num := p.GetNumber()
-		isMerged, _, err := c.c.PullRequests.IsMerged(ctx, org, repo, num)
-		var rateLimitError *github.RateLimitError
-		if errors.As(err, &rateLimitError) {
-			return nil, nil, fmt.Errorf("hit rate limiting")
-		}
-		if err != nil {
-			return nil, nil, fmt.Errorf("error detecting if pr %d is merged or not", num)
-		}
-		if !isMerged {
-			// It means PR has been closed but not merged in
+		if p.GetMergedAt().Equal(time.Time{}) {
```

Comment:
Avoid a call to github API to retrieve the merged state.
Note: `p.GetMerged()` always returns false because `merged` is always empty since github API does not provide it anymore.
See https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#list-pull-requests schema:
```
 "required": [
      "_links",
      "assignee",
      "labels",
      "base",
      "body",
      "closed_at",
      "comments_url",
      "commits_url",
      "created_at",
      "diff_url",
      "head",
      "html_url",
      "id",
      "node_id",
      "issue_url",
      "merge_commit_sha",
      "merged_at",
      "milestone",
      "number",
      "patch_url",
      "review_comment_url",
      "review_comments_url",
      "statuses_url",
      "state",
      "locked",
      "title",
      "updated_at",
      "url",
      "user",
      "author_association",
      "auto_merge"
    ]
```

We use `merged_at` now.